tests/test_Sound.cpp

```diff
@@ -0,0 +1,105 @@
+#include <iostream>
+
+#include <boost/test/unit_test.hpp>
+
+#include <audio/Sound.hpp>
+#include <audio/SoundManager.hpp>
+
+BOOST_AUTO_TEST_SUITE(SoundTests)
+
+BOOST_AUTO_TEST_CASE(createEmptySound) {
+    Sound sound{};
+
+    // buffer and source should be empty
+    BOOST_REQUIRE(sound.buffer == nullptr);
+    BOOST_REQUIRE(sound.source == nullptr);
+}
+
+// @todo Shfil119 implement
+// This test requires assets
+//BOOST_AUTO_TEST_CASE(testBufferIsPlaying) {
+//    Sound sound;
+//    sound.buffer = std::make_unique<SoundBuffer>();
+//    BOOST_REQUIRE(sound.isPlaying() == false);
+
+//    sound.play();
+//    BOOST_REQUIRE(sound.isPlaying() == true);
+
+//    sound.pause();
+//    BOOST_REQUIRE(sound.isPaused() == true);
+
+//    sound.stop();
+//    BOOST_REQUIRE(sound.isStopped() == true);
+//}
+
+BOOST_AUTO_TEST_CASE(checkSettingPosition) {
```

Comment:
Test names should state the expectation explicitly. You already have one expectation per test (good stuff), but the names could be clearer, for example this test could be "sound_sets_openal_source_position" so if it fails in future it's obvious what the test was for before anyone even looks at the code.